### ISSUE TITLE
Remove OJS-specific considerations from pkp-lib

### ISSUE THREAD

**Issue by asmecher** [pkp/pkp-lib#6381]
`PKPSubmissionDAO::getExportable` in pkp-lib references issue IDs, which only apply to OJS. That aspect of the code should be moved into the OJS repository.

https://github.com/pkp/pkp-lib/blob/5a46e84c344ded91d9b635f046b153254c628910/classes/submission/PKPSubmissionDAO.inc.php#L359

(This doesn't necessarily need to be fixed in the RC)

**Comment by NateWr** [comment]
It's hard to imagine a sane way to split the issue part off from the rest of the method without significantly rewriting it. This method powers a bunch of the export grids and is difficult to test thoroughly, so I am nervous to do that rewrite. I opted instead to copy the whole thing into OJS and OPS (without the `issueId`). It was not used in OMP.

I also encountered some errors with the Crossref plugin in OPS where the template file hadn't been updated after the UI refactor, so those are included.

PRs:
https://github.com/pkp/pkp-lib/pull/6384
